go.mod

```diff
@@ -269,7 +269,7 @@ require (
 )
 
 replace (
-	cosmossdk.io/x/upgrade => github.com/celestiaorg/cosmos-sdk/x/upgrade v0.1.0
+	cosmossdk.io/x/upgrade => github.com/celestiaorg/cosmos-sdk/x/upgrade v0.2.0
```

Comment:
Needed to bump x/upgrade to a release that doesn't double increment the app version.